packages/backend-core/src/objectStore/buckets/app.ts

```diff
@@ -51,3 +52,26 @@ export async function getAppFileUrl(s3Key: string) {
     return await objectStore.getPresignedUrl(env.APPS_BUCKET_NAME, s3Key)
   }
 }
+
+export async function enrichPWAImages(
+  images: PWAManifestImage[]
+): Promise<PWAManifestImage[]> {
+  if (!images || images.length === 0) {
```

Comment:
The parameter is not optional, so either this `!images` is not needed or we should make the param optional